payments-core/src/main/java/com/stripe/android/model/PaymentMethod.kt

```diff
@@ -840,6 +856,92 @@ constructor(
         override val type: Type get() = Type.Netbanking
     }
 
+    @Parcelize
+    data class USBankAccount internal constructor(
+        /**
+         * Account holder type
+         *
+         * [us_bank_account.account_holder_type](https://stripe.com/docs/api/payment_methods/object#payment_method_object-us_bank_account-account_holder_type)
+         */
+        @JvmField val accountHolderType: USBankAccountHolderType,
+
+        /**
+         * Account type
+         *
+         * [us_bank_account.account_type](https://stripe.com/docs/api/payment_methods/object#payment_method_object-us_bank_account-account_number)
+         */
+        @JvmField val accountType: USBankAccountType,
+
+        /**
+         * The name of the bank
+         *
+         * [us_bank_account.bank_name](https://stripe.com/docs/api/payment_methods/object#payment_method_object-us_bank_account-bank_name)
+         */
+        @JvmField val bankName: String?,
+
+        /**
+         * Uniquely identifies this particular bank account. You can use this attribute to check
+         * whether two bank accounts are the same
+         *
+         * [us_bank_account.fingerprint](https://stripe.com/docs/api/payment_methods/object#payment_method_object-us_bank_account-fingerprint)
+         */
+        @JvmField val fingerprint: String?,
+
+        /**
+         * Last four digits of the bank account number
+         *
+         * [us_bank_account.fingerprint](https://stripe.com/docs/api/payment_methods/object#payment_method_object-us_bank_account-last4)
```

Comment:
What does [us_bank_account.fingerprint] mean here?

Reply:
My mistake, that should be last4